File: Sistema/API/app/application/Model/AsientoModel.py
from .BaseModel import BaseModel
from ..Shared import db
from ..Shared import ma

class Asiento(BaseModel):
    __tablename__ = 'asiento'
    nombre = db.Column(db.String(128), nullable=True)
    fila = db.Column(db.Integer(), nullable=True)
    columna = db.Column(db.Integer(), nullable=True)
    numero = db.Column(db.Integer(), nullable=True)
    adaptado = db.Column(db.Boolean(), nullable=True)

class AsientoSchema(ma.SQLAlchemyAutoSchema):
    class Meta:
        model = Asiento
        load_instance = True
        sqla_session = db.session

File: Sistema/FE/app/src/app/entities/asiento/asiento.model.ts
export interface IAsiento {
    id?: number;
    nombre?: string;
    fila?: number;
    columna?: number;
    adaptado?: boolean;
}

export class IAsiento implements IAsiento {
    constructor(
        public id?: number,
        public nombre?: string,
        public fila?: number,
        public columna?: number,
        public adaptado?: boolean,
        public disponible?: boolean,
        public seleccionado?: boolean
    ) {
    }
}